### PR TITLE
[wallet] Support CMV signing of 17~20 key template

### PR DESCRIPTION
**Pull request by instagibbs**
Useful in cases where a larger signing set is possible than legacy p2sh(segwit v0?), and for general testing.

### CODE REVIEW

File: src/script/standard.cpp [hunk 1]
```diff
@@ -159,13 +159,15 @@ bool Solver(const CScript& scriptPubKey, txnouttype& typeRet, std::vector<std::v
...
             {   // Single-byte small integer pushed onto vSolutions
                 if (opcode1 == OP_0 ||
-                    (opcode1 >= OP_1 && opcode1 <= OP_16))
-                {
+                    (opcode1 >= OP_1 && opcode1 <= OP_16)) {
                     char n = (char)CScript::DecodeOP_N(opcode1);
                     vSolutionsRet.push_back(valtype(1, n));
-                }
-                else
+                } else if (opcode1 == 0x01 && CScriptNum(vch1, true).getint() > 16 && CScriptNum(vch1, true).getint() <= 20) {
```
Comment (TheBlueMatt) [inline src/script/standard.cpp]: Do we handle scriptnum_error here?

**Review by TheBlueMatt — COMMENTED** [review]
Concept ACK, I suppose.

File: src/script/standard.cpp (hunk 1 above)
Comment (instagibbs) [reply src/script/standard.cpp]: it should be a single byte here, yes? I'm not as knowledgeable on Bitcoin script numerics.

Comment (TheBlueMatt) [reply src/script/standard.cpp]: Hmm? I'm referring to the fact that CScriptNum will throw scriptnum_error here if vch1 is {0x80}.

**Comment by TheBlueMatt** [comment]
Will break the blind casting at https://github.com/bitcoin/bitcoin/blob/62fdf9b07087b80d2142799bdd2324f61483359d/src/script/standard.cpp#L244 (moves to validateaddress' DescribeAddressVisitor in #11403).

**Comment by instagibbs** [comment]
closing as this is broken, and I am not going to fix

**Comment by sipa** [comment]
Doing some refactoring first: #13194

**Pull request closed without merging**